src/time_tracker/timers/handlers.clj

```diff
@@ -42,7 +42,7 @@
   When called with {:date `epoch`}, returns all of the timers 
   created on the same day as `epoch`."
   [request connection]
-  (web-util/validate-request request :timers.handlers/list-all-args)
```

Comment:
I'd handwritten the namespace prefix before. Now it uses simpler syntax. See the changes to my specs for more info.